common/views/components/DefaultPageLayout/DefaultPageLayout.js

```diff
@@ -45,13 +110,13 @@ type AnalyticsProps = {|
 // the new questions we would like ask of our analytics, so this was for a clean slate.
 //
 // `dimension5` is a test dimension. it's `dimension1` on v2
-const Analytics = ({
+export const Analytics = ({
   category,
   seriesUrl,
   positionInSeries,
   contentType
 }: AnalyticsProps) => ([
-  <style key='analytics-1'  dangerouslySetInnerHTML={{ __html: `.async-hide .header__nav{ opacity: 0 !important}`}} />,
+  <style  key='analytics-1' dangerouslySetInnerHTML={{ __html: `.async-hide .header__nav{ opacity: 0 !important}` }} />,
```

Comment:
Extra space before `key`?

Reply:
It's to keep it neat with the style below it.

Reply:
Gross!